app/frontend/styles/application.scss

```diff
@@ -45,6 +45,7 @@ $moj-images-path: "~@ministryofjustice/frontend/moj/assets/images/";
 @import "_diagram";
 @import "_flex-grid";
 @import "_grid";
+@import "_primary-navigation";
```

Comment:
Does it mean we can remove
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/frontend/styles/provider/_all.scss#L6 ?

Reply:
Yep, I'll kill - thanks.